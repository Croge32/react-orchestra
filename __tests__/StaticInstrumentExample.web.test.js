
import React from 'react';
import renderer from 'react-test-renderer';
import StaticInstrument from '../demo/src/components/static-instrument-example';

test('StaticInstrument demo renders correctly', () => {
  // console.log(a);

  const component = renderer.create(
    <StaticInstrument />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
