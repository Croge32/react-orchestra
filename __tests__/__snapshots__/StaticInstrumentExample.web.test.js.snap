exports[`test StaticInstrument demo renders correctly 1`] = `
<div
  style={undefined}>
  <div>
    <span>
      Loading  Instrument ğŸšš ğŸšš ğŸšš
    </span>
  </div>
  <div
    onMouseDown={[Function]}
    onMouseUp={[Function]}>
    <div>
       This is what I want my note to look like ! I can put anything in here.
      <img
        alt="Some pic"
        src="https://s-media-cache-ak0.pinimg.com/originals/36/43/e7/3643e7e8dab9b88b3972ee1c9f909dea.jpg"
        width="80" />
    </div>
  </div>
  <div
    onMouseDown={[Function]}
    onMouseUp={[Function]}>
    <div>
      Another note
    </div>
  </div>
</div>
`;
